js/index.js

```diff
@@ -0,0 +1,28 @@
+// !! IMPORTANT README:
```

Comment:
It would be better to delete instruction comments if you are done with them :)

Reply:
- Please ensure to always state explicit reasons why you requested for a change; it might be obvious to you and not so for others.